test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js

```diff
@@ -570,6 +571,19 @@ describe('Navigation', () => {
         cy.expectPathToBe('/projects/pr2/nav-sync/' + label);
       });
     });
+
+    it('Left navigation title attr not exist', () => {
+      cy.get('.fd-shellbar')
+        .contains('Projects')
+        .click();
+      cy.get('.fd-app__sidebar .fd-nested-list__item')
+        .contains('Project One')
+        .click();
+      cy.get('.fd-nested-list__link')
+        .contains('Miscellaneous2')
+        .parent()
+        .should('have.attr', 'title', 'Miscellaneous2');
```

Comment:
Looking at this test now, I realize this might not capture, cases where **node.tooltipText** and **defaults.tooltipText** are defined. Also the title `Left navigation title attr not exist` is a bit off as the test runs if it does exist.

You would need to reset the Luigi config and then try with **node.tooltipText**  being defined for a node, and **defaults.tooltipText** as well. 

Take a look at this [example](https://github.com/SAP/luigi/blob/7500837126227fc5ea03f3d86c653ab43c62d2b9/test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js#L822), on how you can update luigi configuration while in tests.

Reply:
I changed tests (add new tests which check defaults).